src/ui/components/Library/Navigation/TeamButton.tsx

```diff
@@ -43,25 +45,73 @@ export function TeamButton({
   };
 
   return (
-    <Link
-      href={url}
+    <div
       className={classNames(
-        `${styles.teamRow} group flex flex-row justify-between space-x-2 px-4 py-2 text-left transition duration-200 hover:text-white focus:outline-none`,
-        isSelected ? `${styles.teamRowActive} cursor-auto` : "cursor-pointer"
+        `flex flex-col ${styles.teamRow} `,
+        isSelected ? `${styles.teamRowActive}` : ""
       )}
-      onClick={onClick}
     >
-      <span className="overflow-hidden overflow-ellipsis whitespace-pre">
-        <div className="flex">
-          <LibraryIcon teamType={isTest ? "tests" : "team"} />
-          {label}
+      <Link
+        href={url}
+        className={classNames(
+          `group flex flex-row justify-between space-x-2 px-4 py-2 text-left transition duration-200 hover:text-white focus:outline-none`,
+          isSelected ? `cursor-auto` : "cursor-pointer"
+        )}
+        onClick={onClick}
+      >
+        <span className="overflow-hidden overflow-ellipsis whitespace-pre">
+          <div className="flex">
+            <LibraryIcon teamType={isTest ? "tests" : "team"} />
+            {label}
+          </div>
+        </span>
+        {isNew ? (
+          <div className={"rounded-md bg-primaryAccent px-3 py-0.5 text-xs text-white"}>New</div>
+        ) : null}
+        {showSettingsButton ? <SettingsButton /> : null}
+      </Link>
+      {isSelected ? <TestTeamViews /> : null}
+    </div>
+  );
+}
+
+// We don't currently have a context shared between the navigation
+// and the content of the library UI. Getting the view/teamId this
+// way is pretty clunky, so we should probably lift that state up
+function TestTeamViews() {
+  const [showTestsView] = useLocalStorageUserData("enableTestView");
+  const router = useRouter();
+  const cont = useContext(ViewContext);
+  const view = useGetTeamRouteParams().view;
+  const { teamId } = useGetTeamRouteParams();
+
+  const setView = (view: View) => {
+    pushRoute(router, `/team/${teamId}/${view}`);
+  };
+
+  return (
+    <div className="pl-4 pb-2">
+      <div className="flex flex-col">
+        <div
+          className={`px-4 py-1 ${
+            view === "runs" ? "font-bold" : ""
+          } hover:cursor-pointer hover:font-bold`}
+          onClick={() => setView("runs")}
+        >
+          Runs
```

Comment:
We should only be showing this sub-view for tests suites pages, right? This PR shows it for everything:
https://www.loom.com/share/52299129b46e4430962fac5396e7f588

Also I think we should only show the submenus ("Runs" and "Tests") if the `enableTestView` feature is on, but this PR shows it regardless which is kind of confusing. (Why show only a single option?)

Reply:
This sub-menu is also visible under "Your Library"
<img width="538" alt="Screen Shot 2023-11-15 at 10 44 37 AM" src="https://github.com/replayio/devtools/assets/29597/4fa3b61b-a156-4f45-a0af-5ff90ea536ce">

And clicking these just shows an empty page (because this isn't a test suite workspace).